test-basic/bindingFromParam.js

```diff
@@ -408,7 +409,7 @@ describe('binding from param', function () {
         ];
```

Comment:
@mariusmarklogic  , this test is still failing. Can you please checkin dataFromParam.json file in the test-basic/data folder.